layouts/_default/baseof.html

```diff
@@ -16,10 +16,9 @@
   {{ if isset .Site.Params "twitter" }}
   <meta name="twitter:card" content="summary" />
   <meta name="twitter:title" content="{{ .Title }}" />
-  <meta name="twitter:description" content="{{ if .IsHome }}{{ htmlEscape .Site.Params.description }}{{ else }}{{ htmlEscape .Description }}{{ end }}"
-  />
-  <meta name="twitter:site" content="@{{ .Site.Params.twitter }}" />
-  <meta name="twitter:creator" content="@{{ .Site.Params.twitter }}" />
+  <meta name="twitter:description" content="{{ if .IsHome }}{{ htmlEscape .Site.Params.description }}{{ else }}{{ htmlEscape .Description }}{{ end }}"/>
+  <meta name="twitter:site" content="{{ .Site.Params.twitterSite | default .Site.Params.twitter }}" />
+  <meta name="twitter:creator" content="{{ .Site.Params.twitterCreator | default .Site.Params.twitter }}" />
```

Comment:
Oh ok so I'm seeing what was wrong here. For `twitter:site` and `twitter:creator` we were doing:

```
@https://twitter.com/<your_handle>
```

Which is of course silly.